functions/index.js

```diff
@@ -334,13 +338,25 @@ async function snapshotData(country) {
     headers.push('Row');
     col_labels.push('row');
 
+    const contactFormOptOutIndex = orig_col_labels.findIndex(e => e === '_contact_form_opt_out');
+
     const trimmed_values = real_values.map((row, row_num) => {
       const result = [];
 
+      // Opt out of mail relay if
+      // country's sheet does not have opt out option
+      // OR opt out column has content in it
+      const hasOptedOutOfContact = contactFormOptOutIndex === -1
+        || (Boolean(row[contactFormOptOutIndex]) && row[contactFormOptOutIndex].trim().length > 0);
```

Comment:
How can (Boolean(row[contactFormOptOutIndex]) be true
and row[contactFormOptOutIndex].trim().length be 0?

Reply:
The index === -1 check might have fixed this... I can revisit but originally was checking to make sure that we weren't calling `trim().length` on undefined but we might not need that.

Ultimately I think this field should be explicit as an enum with 'Yes' / 'No'. We can migrate to that by:

1. Replacing the current opt out text with "Yes" but leaving the no's as blank
2. Update functions/index.js to check for presence + value === 'Yes'
3. Update form to write No answers too